itunesSearch/itunesSearch/SearchResultController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  SearchResultController.swift
+//  itunesSearch
+//
+//  Created by Bradley Yin on 8/6/19.
+//  Copyright © 2019 bradleyyin. All rights reserved.
+//
+
+import Foundation
+
+class SearchResultController {
+    let baseURL = URL(string: "https://itunes.apple.com/search?")!
+    
+    var searchResults: [SearchResult] = []
+    
```

Comment:
`performSearch` is a bit long without much vertical breathing space between its internal sections. I would suggest a single newline between the following line numbers: 39/40, 34/35, 29/30,